### DISCUSSION TITLE
Question - how to build examples for Linux?

### DISCUSSION THREAD

**Discussion by seyoungjeong** [Infineon/optiga-trust-m#132] [Q&A]
Sorry for a dumb question. How can I build sources under examples/ for Linux? I'ver searched for a CMakieLists.txt but I can't find.

**Comment by ying-css** [comment]
Hi, @seyoungjeong If you are interested at Linux tools/example for optiga Trust M, please refer to [this](https://github.com/Infineon/linux-optiga-trust-m) Thanks.
